packages/utilities/psammead-storybook-helpers/src/index.test.jsx

```diff
@@ -36,71 +42,178 @@ describe('Psammead storybook helpers', () => {
     beforeEach(() => {
       renderFn = jest.fn();
 
-      boolean.mockClear();
       select.mockClear();
       text.mockClear();
     });
 
-    it('calls the render function with no inputs', () => {
+    it('always calls the render function with a script and direction', () => {
+      const english = {
+        text: 'Could a computer ever create better art than a human?',
+        script: 'latin',
+        dir: 'ltr',
+      };
+      select.mockReturnValueOnce(english);
+
       underTest.inputProvider([], renderFn)();
 
       expect(renderFn).toHaveBeenCalledTimes(1);
-      expect(renderFn).toHaveBeenCalledWith();
+      expect(renderFn).toHaveBeenCalledWith([], 'LATIN SCRIPT OBJECT', 'ltr');
+      expect(select).toHaveBeenCalledTimes(1);
+      expect(text).toHaveBeenCalledTimes(0);
     });
 
-    describe('using free entry text box', () => {
-      beforeAll(() => {
-        mockBooleanResult = false;
-      });
+    it('handles scenario where config is null', () => {
+      const english = {
+        text: 'Could a computer ever create better art than a human?',
+        script: 'latin',
+        dir: 'ltr',
+      };
+      select.mockReturnValueOnce(english);
+
+      underTest.inputProvider(null, renderFn)();
+
+      expect(renderFn).toHaveBeenCalledTimes(1);
+      expect(renderFn).toHaveBeenCalledWith([], 'LATIN SCRIPT OBJECT', 'ltr');
+      expect(select).toHaveBeenCalledTimes(1);
+      expect(text).toHaveBeenCalledTimes(0);
+    });
 
-      it('calls the render function with one input', () => {
-        underTest.inputProvider(['first'], renderFn)();
+    describe('calls the render function with slot default text when displaying english', () => {
+      it('for a single slot', () => {
+        const english = {
+          text: 'Could a computer ever create better art than a human?',
+          script: 'latin',
+          dir: 'ltr',
+        };
+        select.mockReturnValueOnce(english);
+        text.mockImplementation((_, displayText) => displayText);
```

Comment:
What does the `_` do in this syntax?

Reply:
Nothing special; it's a convention I've encountered in other teams to show that the parameter is ignored. If it's not clear/used here I've named it as usual